src/lib/rpcApi.js

```diff
@@ -11,6 +11,29 @@ export class RpcApi {
   }
 
   call(name, params, headers) {
+    return this.jsonPost(name, params, headers);
+  }
+
+  /**
+   * This call analyses JSON RPC response.
+   * It returns promise which resolves whether 'result' field found
+   * or reject in case 'error' field found.
+   *
+   * @returns {Promise}
+   */
+  call2(name, params, headers) {
```

Comment:
I prefer `call2`. I guess we'll have a later PR to replace all the `call` with `call2`